test/test_playbooks/organization.yml

```diff
@@ -10,6 +10,8 @@
       organization_state: absent
 - hosts: tests
   gather_facts: false
+  environment:
+    XDG_CACHE_HOME: "{{ lookup('env', 'FAM_TEST_APYPIE_CACHE_DIR') }}"
```

Comment:
this is needed as I want to override the cache folder only for the tests hosts and one can't set env vars in the inventory :(